fdbserver/workloads/FuzzApiCorrectness.actor.cpp

```diff
@@ -1213,6 +1213,9 @@ struct FuzzApiCorrectnessWorkload : TestWorkload {
 	};
 
 	static void addTestCases() {
+		// this group passes on its own.
+		// TestGet + TestSet passes.
+
```

Comment:
can delete these comments.  these were notes to myself.  (and also the ones below)